flashinfer/gemm.py

```diff
@@ -721,14 +723,23 @@ def gemm_fp8_nt_groupwise(
         Column-major input tensor shape (n, k), fp8 e4m3 or fp8 e5m2.
 
     a_scale: torch.Tensor
-        Column-major scale tensor for a, shape (ceil_div(k, k_granularity), ceil_div(m, m_granularity)).
+        Column-major scale tensor for a, shape ``(m, k // block_size)`` if scale_major_k is ``True``
+        otherwise ``(k // block_size, m)``.
 
     b_scale: torch.Tensor
-        Row-major scale tensor for b, shape (ceil_div(k, k_granularity), ceil_div(n, n_granularity)).
+        Row-major scale tensor for b, shape ``(n // block_size, k // block_size)`` if scale_major_k is ``True``
+        otherwise ``(k // block_size, n // block_size)``.
 
     scale_granularity_mnk: Tuple[int, int, int]
         The granularity of the scale tensor, (m_granularity, n_granularity, k_granularity).
 
+    scale_major_k: bool
+        Whether scale tensors are k-major layout.
```

Comment:
better to explain what's `k-major` (and the alternative `mn-major`) because it can be confusing to new users.

The name `scale_major_k` is also confusing, can we use`scale_major_mode: str = "MN" or "K"` instead?